project/__init__.py

```diff
@@ -16,9 +17,12 @@ def create_app(config):
     def home():  # pylint: disable=unused-variable
         """Home page."""
         form = QueryForm()
+        query_class = Query()
```

Comment:
We probably don't need to instantiate a Query class every time the user visits the home page. Perhaps we can make Query a singleton class: http://python-3-patterns-idioms-test.readthedocs.io/en/latest/Singleton.html. The first method in the article should suffice (using __OnlyOne and instance). The idea is that we don't want to initialize the API handlers every time so we should initialize it once and subsequent calls to Query just reuse the instance variable.

Reply:
That being said, you can make this change in the next pull request as I don't want to stall this current pull request too much.

Reply:
Yeah, I just used this for testing purposes, the people using the query class would handle this (NLP ppl).